upup/models/cloudup/_aws/topologies/hybrid1.yaml

```diff
@@ -0,0 +1,53 @@
+{{ if IsTopologyHybrid1 }}
```

Comment:
You _could_ use a tag here.  Still think code is easier still (and would avoid some of the repetition) ;-)